### PR TITLE
Create topic for maintenance windows

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3072

Considered putting this in a new topic called `Configure cluster maintenance window (GCP only)` in the same doc. LMK if that's better!

### CODE REVIEW

File: astro/create-dedicated-cluster.md [hunk 1]
```diff
@@ -61,6 +61,16 @@ Dedicated clusters offer the self-service convenience of a fully managed service
    
 4. Click **Create cluster**. After Astro finishes creating the cluster, users in your Organization can select the cluster when they [create a Deployment](create-deployment.md). 
 
+:::info Configure cluster maintenance windows
+
+All GCP dedicated clusters are subscribed to the [GKE regular release channel](https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels), meaning that Google automatically upgrades the cluster and its nodes whenever an upgrade is available.
+
+After you create a GCP cluster, you can control when these upgrades happen by requesting a [maintenance window](https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions#maintenance-window) for the cluster. Maintenance windows determine when and how Google updates our cluster. You can use maintenance windows to ensure that upgrades don't happen while critical DAGs are running on your cluster.
```
Comment (lzdanski) [inline astro/create-dedicated-cluster.md:68]:
```suggestion
After you create a GCP cluster, you can control when these upgrades happen by requesting a [maintenance window](https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions#maintenance-window) for the cluster. Maintenance windows determine when and how Google updates your cluster. You can use maintenance windows to ensure that upgrades don't happen while critical DAGs are running on your cluster. 
```

**Review by lzdanski — APPROVED** [review]
typo!

**Comment by chomipi88** [comment]
My gut tells me we should call out the below items as well.

1. There are [considerations/limitations](https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions#maintenance-window) that must be adhered to when configuring a maintenance window.

2. This maintenance window does not exempt Astro from pushing application/platform updates, bug and security fixes. These will continue to get rolled out weekly/bi-weekly when updates are made available for release.

**Comment by chomipi88** [comment]
oh and this capability also applies to Hybrid!

File: astro/create-dedicated-cluster.md [hunk 2]
```diff
@@ -61,6 +61,16 @@ Dedicated clusters offer the self-service convenience of a fully managed service
...
 
+:::info Configure cluster maintenance windows
+
+All GCP dedicated clusters are subscribed to the [GKE regular release channel](https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels), meaning that Google automatically upgrades the cluster and its nodes whenever an upgrade is available.
+
+After you create a GCP cluster, you can control when these upgrades happen by requesting a [maintenance window](https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions#maintenance-window) for the cluster. Maintenance windows determine when and how Google updates your cluster. You can use maintenance windows to ensure that upgrades don't happen while critical DAGs are running on your cluster. 
+
+To set a maintenance window, first choose a maintenance window time and read through the [maintenance window considerations](https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions#considerations) to make sure that the time is optimized for your cluster. Then, contact [Astronomer Support](https://cloud.astronomer.io/open-support-request) and provide your cluster ID and desired support window configuration.
```
Comment (jwitz) [inline astro/create-dedicated-cluster.md:70]:
```suggestion
To set a maintenance window, first choose a maintenance window time and read through the [maintenance window considerations](https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions#considerations) to make sure that the time is optimized for your cluster. Then, contact [Astronomer Support](https://cloud.astronomer.io/open-support-request) and provide your cluster ID and desired support window.
```

**Pull request merged**